jbmc/regression/jbmc/must-not-throw-annotation/Test.java

```diff
@@ -0,0 +1,17 @@
+public class Test {
+
+  @org.cprover.MustNotThrow
+  public static void must_not_throw() {
```

Comment:
Nit: `mustNotThrow`

Reply:
It's a class name, so usually it would start with a capital? For example, `@org.junit.Test`

Reply:
Sorry should have been less ambiguous - your method under test should be camelCased